string message_id
string agent_id_from
string agent_id_to
string message_type
string params